src/TraceEvent/Symbols/SymbolReader.cs

```diff
@@ -979,7 +979,11 @@ internal bool GetPhysicalFileFromServer(string serverPath, string pdbIndexPath,
                         {
                             m_log.WriteLine("FindSymbolFilePath: In task, sending HTTP request {0}", fullUri);
 
-                            var responseTask = HttpClient.GetAsync(fullUri, HttpCompletionOption.ResponseHeadersRead);
+                            // Tell the symbol server that we support MSFZ symbols
+                            var request = new HttpRequestMessage(HttpMethod.Get, fullUri);
+                            request.Headers.Add("Accept", "application/msfz0");
```

Comment:
When a symbol file is downloaded, there is a callout that can be registered.  PerfView registers this callout and implements it in PerfView.App.CacheInLocalSymDir.  You will also need to update App.cs line 960 to include the msfz0 subdirectory change if the file is an msfz format file.  You can do a quick check on `pdbPath` to see if the last subdirectory right before the actual file is "msfz0".